src/views/Login/PrivateKey.js

```diff
@@ -1,80 +1,50 @@
-import Maybe from 'folktale/maybe';
-import React from 'react';
-import { Button } from '../../components/old/Base';
-import {
-  RequiredInput,
-  InnerLabel,
-  InputCaption,
-} from '../../components/old/Base';
-import { Row, Col, H1 } from '../../components/old/Base';
+import { Just, Nothing } from 'folktale/maybe';
+import React, { useState } from 'react';
 
-import { EthereumWallet } from '../../lib/wallet';
-import { compose } from '../../lib/lib';
-import { withWallet } from '../../store/wallet';
+import View from '../../components/View';
```

Comment:
we can actually get rid of these relative imports now that we look for modules in `src`: `import View from 'components/View'` should work perfectly. likewise across these files.